src/flib/pio_kinds.F90

```diff
@@ -44,11 +33,4 @@ module pio_kinds
 !> Byte size of the MPI_OFFSET type.
    integer, parameter, public :: PIO_OFFSET_KIND=MPI_OFFSET_KIND
 
-!EOP
-!BOC
-!EOC
```

Comment:
Should I put these back in? Or remove them everywhere (my desire if they are no longer used)?